src/app/shared/issue-tables/issue-tables.component.ts

```diff
@@ -95,10 +95,9 @@ export class IssueTablesComponent implements OnInit, AfterViewInit {
   }
 
   markAsResponded(issue: Issue) {
-    this.issueService.updateIssue(<Issue>{
-      ...issue,
-      status: STATUS.Done
-    }).subscribe((updatedIssue) => {
+    const newIssue = issue.clone(this.phaseService.currentPhase);
```

Comment:
Can I clarify how the changes to the issue update logic relate with the session ID implementation? Isn't the hidden data already a property of the issue?

Reply:
It was a technical debt introduced at the start of the project. Previously, the update was done by create a new typed object instead of a Class. By creating a typed object, we are erasing the Issue's method associated to it. As a result, we will not be able to make use of Issue's method, which in this case is `createGithubIssueDescription`.